kura/org.eclipse.kura.core.cloud/OSGI-INF/cloud.xml

```diff
@@ -76,4 +76,5 @@
               unbind="unsetJsonMarshaller"/>
    <property name="kura.ui.service.hide" type="Boolean" value="true"/>
    <reference bind="setTamperDetectionService" cardinality="0..n" interface="org.eclipse.kura.security.tamper.detection.TamperDetectionService" name="TamperDetectionService" policy="dynamic" unbind="unsetTamperDetectionService"/>
+   <reference bind="setNetworkStatusService" cardinality="0..1" interface="org.eclipse.kura.net.status.NetworkStatusService" name="NetworkStatusService" policy="dynamic" unbind="unsetNetworkStatusService"/>
```

Comment:
Please format this like the other references (the reference above needs cleanup as well)